src/pages/why.tsx

```diff
@@ -0,0 +1,412 @@
+                                        disappear overnight.
+                                    </li>
+                                    <li>We don’t screw you on contract terms like auto-renewal.</li>
+                                    <li>
+                                        You can use all our stuff monthly until you want to lock things in with us. We
+                                        don’t pressure people.
+                                    </li>
+                                    <li>
+                                        The way we write our content. <Link href="/posts">Just read it</Link> and you’ll
+                                        get what I mean. It’s more honest and not marketing for the sake of it - it
+                                        actively helps developers build successful products. That’s our mission.
+                                    </li>
+                                    <li>
+                                        PostHog has zero intention of selling our business. We want to see just how
+                                        crazy huge this gigantic software stack can get - and we think that it will
+                                        reach at least $100bn in value. We’ll be around and fighting for a long, long
+                                        time. It’s our life’s work.
+                                    </li>
+                                    <li>
+                                        You can also see how our entire company operates - you can see our{' '}
+                                        <Link href="/handbook/growth/sales/overview">manual for our salespeople</Link>,{' '}
+                                        <Link href="/handbook/growth/marketing">marketing team</Link>, and so on. You
+                                        can even <Link href="/handbook/people/compensation">see what they’re paid</Link>{' '}
+                                        - it’s all in our public handbook. Even the contentious stuff – there’s a reason
+                                        most companies hide this stuff!
+                                    </li>
+                                    <li>So many hedgehogs.</li>
+                                </ul>
+
+                                <p>
+                                    The reason we uniquely love doing all these sorts of things is that{' '}
+                                    <strong>
+                                        we grow because of our reputation on the internet whereas every single
+                                        competitor grows by salespeople.
+                                    </strong>{' '}
+                                    We are very, very proudly inbound only. This aligns us with customers. Long term,
+                                    that is what wins.
```

Comment:
```suggestion
                                    We do all this because <strong>we don't rely on salespeople to grow</strong>.
                                 </p>
                                 <p>   
                                    We are <strong>very proudly inbound only</strong>. We won't cold email you. We won't insist you jump on a quick call, unless you want to. 
                                  </p>
                                  <p>  
                                    This aligns us with customers. Long term, that is what wins.
```